airbyte-api/build.gradle

```diff
@@ -103,7 +101,7 @@ task generateApiDocs(type: GenerateTask) {
     ]
 
     doLast {
-        def target = file(rootProject.file("docs/api"))
+        def target = file(rootProject.file("docs/api/generated-api-html"))
```

Comment:
I searched and did not find any references to the old link - let me know if I missed anything!